tests/extractors/test_command.py

```diff
@@ -36,7 +36,7 @@ def test_command_execution(tmpdir: Path):
     outdir = PosixPath(tmpdir)
     command = Command("sh", "-c", "> {outdir}/created")
     res = TaskResult()
-    assert command.extract(Path("foo"), outdir, res)
+    command.extract(Path("foo"), outdir, res)
```

Comment:
I'd merge these changes back to the previous commit introducing the boolean return value